crit/explore.go

```diff
@@ -373,3 +373,110 @@ func (c *crit) ExploreRss() ([]*RssMap, error) {
 
 	return rssMaps, nil
 }
+
+// Sk represents the sockets associated with a single process
+type Sk struct {
+	PId     uint32    `json:"pId"`
+	Sockets []*Socket `json:"sockets"`
+}
+
+// Socket represents a single socket
+type Socket struct {
+	Fd       uint32 `json:"fd"`
+	FdType   string `json:"fdType"`
+	Family   string `json:"family,omitempty"`
```

Comment:
Would it make sense to also add `Protocol`? For example, something like the following: 
```Go
Protocol string `json:"proto,omitempty"`
```
```Go
socket.Protocol = getProto(isk.GetProto())
```
```Go
func getProto(proto uint32) string {
	switch proto {
	case syscall.IPPROTO_ICMP:
		return "ICMP"
	case syscall.IPPROTO_ICMPV6:
		return "ICMPV6"
	case syscall.IPPROTO_IGMP:
		return "IGMP"
	case syscall.IPPROTO_RAW:
		return "RAW"
	case syscall.IPPROTO_TCP:
		return "TCP"
	case syscall.IPPROTO_UDP:
		return "UDP"
	case syscall.IPPROTO_UDPLITE:
		return "UDPLITE"
	default:
		return ""
	}
}
```

Reply:
Ah so protocol is shortened to `proto`. I mistook that to be the descriptor for the protobuf definition :sweat_smile: